src/main/java/uk/gov/hmcts/reform/draftstore/endpoint/v2/DraftStoreEndpoint.java

```diff
@@ -27,6 +28,12 @@
 import static org.springframework.web.bind.annotation.RequestMethod.POST;
 import static uk.gov.hmcts.reform.draftstore.domain.SaveStatus.Updated;
 
+@APIDeprecated(
+    name = "/api/v2/draft endpoint",
+    expiryDate = "2018-01-31",
```

Comment:
Can we make it `2017-11-31` and then extend if needed?

Reply:
doesn't really make sense to go that way around. we'll set a long date and then reduce it if we can. getting 2 deployment slots in November given activity on the program and no CD at the moment seems unlikely